test/puppetlabs/trapperkeeper/services/authorization/authorization_core_test.clj

```diff
@@ -48,109 +51,139 @@
             params [single-query-param multi-query-param]]
       (let [rule (merge-with merge base allow deny params)]
         (when (or allow deny)
-          (is (= rule (validate-auth-config-rule! rule))))))))
+          (is (= rule (validate-auth-config-rule! rule))))
+        (let [parsed (config->rule rule)]
+          (is (= (:sort-order parsed) (:sort-order rule)))
+          (is (= (:name parsed) (:name rule)))
+          (is (false? (contains? parsed :match-request))))))))
+
+(def testrule
+  "A valid rule definition as it would be found in the configuration file."
+  {:name "name"
+   :sort-order 1
+   :allow "allow"
+   :match-request {:path "/path" :type "path"}})
```

Comment:
I refactored all the tests below to do `assoc`,`dissoc` on this rather than repeating explicit maps